src/store/trees/trees.js

```diff
@@ -79,6 +79,8 @@ export default {
 
 		interactiveBlockClasses: ({ui}, getters, rootState) => lodash.pickBy(ui.blockClasses, (value, key) => value.is_interactive),
 
+    blockClassFromBlockType: ({ui}, getters, rootState) => (blockType) => lodash.values(lodash.pickBy(ui.blockClasses, (value, key) => value.type === blockType))[0],
```

Comment:
I'm personally not a fan of this type of abstraction, since we're returning a closure, it's going to generate a new function definition every time it's called, it makes it unobvious at the consumer call sites as to what is data and what needs to be invoked, and additionally: this function then needs to scan all block classes in every call on every redraw iteration at the template level where we're calling this from! Depending on how much data we're dealing with, this pattern can get smelly quite quick.

From my perspective this would be happier living as a standalone function at the module level that we can import and use. Otherwise, I'd love to see this rather as an index, like:
```
blockClassesByBlockType: ({ui}, getters, rootState) => keyBy(ui.blockClasses, 'type')

// later ...

const cls = blockClassesByBlockType[type]
```

Reply:
- [ ] @safydy to remove parameterized getter in favour of either a module-level function or a pre-indexed computed getter over `blockType`.

Reply:
removed `blockClassFromBlockType` usage